app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
```

Comment:
- [ ] If you check the `comments` table,  you'll discover that one vital field is missing: post_id which makes it impossible to know the post(post_id) user(default user_id should be modified to author_id as in ERD) will be commenting on it. I'll kindly suggest you fix this issue so that all this information is retrievable :+1: . After fixing, modify this line of code as pointed out above :arrow_up:

Reply:
I am not sure what you mean by that. Can you please clarify this one as well? As much as I can see my comments table already has a post_id.

Reply:
@Muhammad0602, below is the comments table from the db. It seems you didn't add post as a reference in this table
```

  create_table "comments", force: :cascade do |t|
    t.text "text"
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
  end
```

Reply:
> where did you get this? Here is my table and it does have the post_id
> 
> ```
> class CreateComments < ActiveRecord::Migration[7.0]
>   def change
>     create_table :comments do |t|
>       t.references :user, null: false, foreign_key: true
>       t.references :post, null: false, foreign_key: true
>       t.text :text
> 
>       t.timestamps
>     end
>   end
> end
> ```

Oh, I see, but I don't know why the comments table didn't update with the required field(post_id) after migrating